.github/workflows/test-pr.yml

```
@@ -5,32 +5,32 @@ on:
-    if: |
-      github.event.issue.pull_request && 
-      startsWith(github.event.comment.body, '/make ')
+    if: "github.event.issue.pull_request && \nstartsWith(github.event.comment.body, '/make ')\n"
     runs-on: ubuntu-latest
     timeout-minutes: 120
-
     steps:
       - name: Extract and validate Make target
+
         id: extract-target
         run: |
```

actionlint (1.7.12)

[error] "github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)